Source/Core/Common/Timer.h

```diff
@@ -26,15 +26,20 @@ class Timer
 	static void RestoreResolution();
 	static u64 GetTimeSinceJan1970();
 	static u64 GetLocalTimeSinceJan1970();
+	// Returns a timestamp with decimals for precise time comparisons
 	static double GetDoubleTime();
 
 	static std::string GetTimeFormatted();
+	// Formats a timestamp from GetDoubleTime() into a date and time string
+	static std::string GetDateTimeFormatted(double time);
 	std::string GetTimeElapsedFormatted() const;
 	u64 GetTimeElapsed();
 
 	static u32 GetTimeMs();
 	static u64 GetTimeUs();
 
+	static const int DOUBLE_TIME_OFFSET = (38 * 365 * 24 * 60 * 60);
```

Comment:
And a comment here that the value itself isn't really significant at all; but arbitrarily chosen to make the number being stored a lot smaller (by 38 years to be exact :3 )